_tools/release_docs/api-reference-process.go

```diff
@@ -97,12 +98,24 @@ func nitFix(filename string) error {
 	return nil
 }
 
-//move the definitions.html and operations.html to _includes
-func moveHTML(filePath string, fileName string, outputDir string) error {
+// move the definitions.html and operations.html to _includes, with name like
+// extensions-v1beta1-definitions.html.
+func moveHTML(filePath string, outputDir string) error {
+	dirs := strings.Split(filePath, "/")
+	var i int
+	for i = 0; i < len(dirs); i++ {
+		if dirs[i] == "api-reference" {
+			break
+		}
+	}
+	if i == len(dirs) {
+		return fmt.Errorf("failed to find fold \"api-reference\"\n")
+	}
+	fileName := strings.Join(dirs[i+1:], "-")
```

Comment:
Sorry can you explain how does the name end up being extensions-v1beta1-definitions.html?
Whats the name for the other v1 file?